README.md

```diff
@@ -73,23 +73,38 @@ project.languages      #=> { "Ruby" => 119387 }
 #### Git Repository
 
 A repository's languages stats can also be assessed from the command line using the `github-linguist` executable.
-Without any options, `github-linguist` will output the breakdown that correlates to what is shown in the language stats bar.
-The `--breakdown` flag will additionally show the breakdown of files by language.
+Without any options, `github-linguist` will output the language breakdown by percentage and file size.
 
 ```bash
-cd /path-to-repository/
+cd /path-to-repository
 github-linguist
 ```
 
 You can try running `github-linguist` on the root directory in this repository itself:
 
+```console
+$ github-linguist
+66.84%  264519     Ruby
+24.68%  97685      C
+6.57%   25999      Go
+1.29%   5098       Lex
+0.32%   1257       Shell
+0.31%   1212       Dockerfile
+```
+
+##### Additional options - Breakdown
```

Comment:
Having multiple subheadings prefixed by `Additional options -` looks messy, IMHO. What about an intermediate heading for grouping/outline purposes? Something like:

~~~markdown
##### Additional options

###### `--breakdown`
…

###### `--json`
…
~~~

Reply:
That works for me. Updated.